add note for using Caplin as beacon chain endpoint (#1068)
OffchainLabs/arbitrum-docs: arbitrum-docs/node-running/how-tos/running-a-full-node.mdx

  - If the chain is running [ArbOS 20](/node-running/reference/arbos-software-releases/arbos20.mdx), additionally use the parameter `--parent-chain.blob-client.beacon-url=<Layer 1 Ethereum Beacon RPC URL>` for consensus layer. You can find a list of beacon chain RPC providers [here](/node-running/reference/ethereum-beacon-rpc-providers.mdx).
  - It must provide a standard layer 1 node RPC endpoint that you run yourself or from a node provider.
  - Note: this parameter was called `--l1.url` in versions prior to `v2.1.0`

>   - Note: the ethereum consensus layer client Caplin is not compatible with nitro node due to some response type format reasons

Reviewer: ```suggestion
  - Note: the Ethereum consensus layer client Caplin is not compatible with Nitro node software due to an issue with response type formatting.
```
